src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,111 @@
+import React, {
+  Dispatch,
+  ReactElement,
+  ReactNode,
+  SetStateAction,
+} from "react";
+import styled, { css, useTheme } from "styled-components";
+import { Menu } from "@headlessui/react";
+import { IconDirection } from "../Icon/Direction";
+import { Text } from "../../typography";
+import { Button } from "../Button";
+import Icon from "../Icon";
+
+export interface DropdownOption {
+  value: string;
+  label: string;
+}
+
+export interface DropdownProps {
+  buttonContent: ReactNode;
+  options: DropdownOption[];
+  selected?: DropdownOption;
+  setSelected: Dispatch<SetStateAction<DropdownOption | undefined>>;
+}
+
+export const Dropdown = ({
+  buttonContent,
+  options,
+  selected,
+  setSelected,
+}: DropdownProps): ReactElement | null => {
```

Comment:
there is no case where this actually returns a null value right?

Reply:
Correct, initially there was, but then I refactored and forgot to remove this. Nicely spotted!

Removed with https://github.com/Major-Eth/yearn-web-lib/pull/28/commits/c200e2d9039607d732da51042430d64c20d3f073